zebra/zebra_pw.c

```diff
@@ -506,6 +508,155 @@ DEFUN (show_pseudowires,
 	return CMD_SUCCESS;
 }
 
+static void vty_show_mpls_pseudowire_detail(struct vty *vty)
+{
+	struct zebra_vrf *zvrf;
+	struct zebra_pw *pw;
+	struct route_entry *re;
+	struct nexthop *nexthop;
+
+	zvrf = vrf_info_lookup(VRF_DEFAULT);
+	if (!zvrf)
+		return;
+
+	RB_FOREACH (pw, zebra_pw_head, &zvrf->pseudowires) {
+		char buf_nbr[INET6_ADDRSTRLEN];
+		char buf_nh[100];
+
+		vty_out(vty, "Interface: %s\n", pw->ifname);
+		inet_ntop(pw->af, &pw->nexthop, buf_nbr, sizeof(buf_nbr));
+		vty_out(vty, "  Neighbor: %s\n",
+			(pw->af != AF_UNSPEC) ? buf_nbr : "-");
+		if (pw->local_label != MPLS_NO_LABEL)
+			vty_out(vty, "  Local Label: %u\n", pw->local_label);
+		else
+			vty_out(vty, "  Local Label: %s\n", "-");
+		if (pw->remote_label != MPLS_NO_LABEL)
+			vty_out(vty, "  Remote Label: %u\n", pw->remote_label);
+		else
+			vty_out(vty, "  Remote Label: %s\n", "-");
+		vty_out(vty, "  Protocol: %s\n",
+			zebra_route_string(pw->protocol));
+		if (pw->protocol == ZEBRA_ROUTE_LDP)
+			vty_out(vty, "  VC-ID: %d\n", pw->data.ldp.pwid);
```

Comment:
needs to use the `%u` format specifier instead (didn't catch this earlier)

Reply:
should have caught that too, fixed